src/features/PesterTests.ts

```diff
@@ -47,6 +49,7 @@ export class PesterTestsFeature implements IFeature {
             ],
             internalConsoleOptions: "neverOpen",
             noDebug: !runInDebugger,
+            createTemporaryIntegratedConsole: settings.debugging.createTemporaryIntegratedConsole,
```

Comment:
Should the setting only be used when runInDebugger is true?  I could go either way

Reply:
I think there is benefit in being able to run tests in a fresh session.

Reply:
Maybe if "runInDebugger" is false we can use a different title?  That was short-sighted on my part

Reply:
Or `[TEMP]` just to be extra clear, but anything works

Reply:
I wish the VSCode folks would make this dropdown a tad wider:
![image](https://user-images.githubusercontent.com/5177512/36634835-27eb17ce-1968-11e8-966b-f76fb2e21c38.png)
What do you think?